versioning/versions/versions.go

```diff
@@ -26,6 +26,7 @@ type Version struct {
 	Tags         []string
 	From         string
 	Cmd          string
+	TemplateArgs map[string]string
```

Comment:
We can call this `Values` to avoid redundant "template" when this is used from the template. Essentially, we'll have

```
{{- $custom := index .Values "debian_repo" }}
```

instead of

```
{{- $custom := index .TemplateArgs "debian_repo" }}
```